scripts/online_roach_powers.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python
```

Comment:
I'm not sure I get the name of this script. Something with a verb would be better, I think. How about "mc_publish_summary.py"? I also think it's important to namespace the scripts (e.g. use the `mc_` prefix) to maintain some level of organization; our `bin/` directories are rapidly accumulating a whole mess of scripts from a whole bunch of subsystems, which makes it hard to remember names and meanings.

Reply:
renamed to mc_publish_sys_levels.py